src/main/java/com/puppycrawl/tools/checkstyle/checks/FinalParametersCheck.java

```diff
@@ -68,13 +68,71 @@
  * &lt;module name=&quot;FinalParameters&quot;/&gt;
  * </pre>
  * <p>
+ * Example :
+ * </p>
+ * <pre>
+ * public class Point {
+ *       private int m,n;
+ *       private String s;
+ *       public Point(){ //ok
+ *       s = "ABC";
+ *       }
+ *       public Point(final int m) { // ok
+ *           this();
```

Comment:
This Check do not care about body content. so please make all metods single line
`public Point(final int m) { }`